src/components/__snapshots__/ErrorMessage.test.tsx.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`renders correctly 1`] = `
-<div>
-  Content goes here!
-</div>
+<span
+  className="govuk-error-message undefined lbh-error-message"
```

Comment:
We don't want `undefined` in here. See the [`classnames` package](https://www.npmjs.com/package/classnames).